pkg/aspect/config/quotas.proto

```diff
@@ -20,7 +20,16 @@ option go_package="config";
 
 // Configures a quotas aspect.
 message QuotasParams {
-    // A list of names of istio.mixer.v1.config.descriptor.QuotaDescriptors determining the
-    // set of quotas that will be checked for any Quota call.
-    repeated string quota_descriptor_names = 1;
+    message Quota {
+        // Must match the name of some quota_descriptor in the global config.
+        string descriptor = 1;
```

Comment:
should this be `descriptor_name` ?  this should also help avoid the unsightly `Descriptor_` in the generated code.